### PR TITLE
docs(trg1): align docs directory structure

### PR DESCRIPTION
**Pull request by typecastcloud**
## Description

- move existing documents to folder structure in trg 1. eclipse-tractusx/sig-release#805
- create README.md with references
- rename existing files with clear relation to arc42 standard 

## Why

Follow TRG 1

## Issue

Closes: eclipse-tractusx/ssi-authority-schema-registry#58
Refs: eclipse-tractusx/sig-release#805

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/ssi-authority-schema-registry/blob/main/docs/technical-documentation/dev-process/How%20to%20contribute.md)
- [ ] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [ ] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have added copyright and license headers, footers (for .md files) or files (for images)

### CODE REVIEW

File: docs/api/Org.Eclipse.TractusX.SsiAuthoritySchemaRegistry.Service.postman_collection.json
Comment (evegufy) [inline docs/api/Org.Eclipse.TractusX.SsiAuthoritySchemaRegistry.Service.postman_collection.json:1]: quite recently there was an api directory introduced: https://github.com/Cofinity-X/entry-ssi-authority-schema-registry/blob/edc06c1c739bdbc679821a8acea79791b2f167a0/docs/api/asr-service.yaml
As part of https://github.com/eclipse-tractusx/ssi-authority-schema-registry/pull/66
So, the asr-service.yaml file should be in this directory.

@Phil91 where should be put the postman collection? Maybe api/requests? Are we still maintaining the collection in addition to the api spec?

File: docs/README.md
Comment (evegufy) [inline docs/README.md:1]: I think the docs/README.md is intented to provide a short overview about the the available documentation and shouldn't be be limited to presenting the arc42 structure, but great that you added a glossary, maybe you want to add it as a separate file under /architecture?

**Review by evegufy — CHANGES_REQUESTED** [review]
Looks really good so far, just some comments

File: docs/README.md
Comment (typecastcloud) [reply docs/README.md:1]: I didn't want to repeat anything already written in the linked document. Do you think the README should contain a list of what the document already contains or what it is supposed to contain?

File: docs/api/Org.Eclipse.TractusX.SsiAuthoritySchemaRegistry.Service.postman_collection.json
Comment (typecastcloud) [reply docs/api/Org.Eclipse.TractusX.SsiAuthoritySchemaRegistry.Service.postman_collection.json:1]: I moved the postman collection into docs/api/postman folder I created to make it more clear.

File: docs/README.md
Comment (typecastcloud) [reply docs/README.md:1]: Hi @evegufy, thank you for the feedback and link to the example.

I reworked the structure of all documents I could find and renamed them in arc42 format.

There are some documents missing for full compliance with arc42.

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+This repository provides the backend code for the SSI Authority & Schema Registry (SSI ASR).
+
+The following table links you to the respective documentations.
+
+| Documentation                                                    | Purpose                                                         |
+|------------------------------------------------------------------|-----------------------------------------------------------------|
+| [Arc42](architecture/Index.md)                                   | Architecture Documentation for the SSI Authority & Schema Registry (SSI ASR).             |
+| [How to contribute](./admin/dev-process/How%20to%20contribute.md)| Information relevant, if you contribute in ssi authority schema registry development.   |
+| [Administration Guide](admin/Admin_Guide.md)                     | Information relevant, if you want to use the ssi authority schema registry application. |
+| [Interface Documentation](api/Interface_Doc.md)                  | Information about the interfaces.                               |
```
Comment (evegufy) [inline docs/README.md:12]:
```suggestion
| [API Documentation](api/API_Doc.md)                  | Information about the endpoints.                               |
```

File: docs/api/Interface_Doc.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Interface Documentation
```
Comment (evegufy) [inline docs/api/Interface_Doc.md:1]:
```suggestion
# API Documentation
```

File: docs/api/Interface_Doc.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+This document provides information on the interfaces. It more or less links you to the relevant sections in other
+documentations.
```
Comment (evegufy) [inline docs/api/Interface_Doc.md:4]:
```suggestion
This document provides information on the endpoints.
```

File: docs/api/Interface_Doc.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+## Overview of Data Exchange Interfaces
+
+The [Building Block View](../architecture/05_building_block_view.md) provides an overview.
```
Comment (evegufy) [inline docs/api/Interface_Doc.md:8]:
```suggestion
```

**Review by evegufy — CHANGES_REQUESTED** [review]
Looks very good, just some small comments still

**Comment by typecastcloud** [comment]
> Looks very good, just some small comments still

Good feedback @evegufy. Less confusing. Resolved.

**Review by evegufy — APPROVED** [review]
thank you! could you please still change the PR title `docs(trg1): align docs directory structure`
I find the issue number sig805 very useful in the PR description and branch name, but not so much in the title of the PR.

@Phil91 would you like to review as well?

**Comment by typecastcloud** [comment]
> thank you! could you please still change the PR title `docs(trg1): align docs directory structure` I find the issue number sig805 very useful in the PR description and branch name, but not so much in the title of the PR.
> 
> @Phil91 would you like to review as well?

Title updated and sig release issue linked/referenced in description.

**Review by Phil91 — APPROVED** [review]
looks really really good!

**Pull request merged**